StringEvaluate.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Stack;
+
+
+public class StringEvaluate {
+
+	public static boolean evaluate(String input){
+		
+		if (input == null || input == ""){
+			return true;
+		}
+		else if(input.length() == 1){
+			return false;
+		}
+		
+		Stack<Character> myStack = new Stack<>();
+		char c , check;
+		
+		for(int i = 0 ; i < input.length() ; i++){
+			
+			c = input.charAt(i);
+			
+			if(c == '(' || c == '{' || c == '['){
+				myStack.push(c);
+			}
+			
+			if(c == ')' || c == '}' || c == ']'){
+				
+				if( myStack.isEmpty() ){
+					return false;
+				}
+				check = myStack.pop();
+				
+				if( !(check == '(' && c == ')' || check == '{' && c == '}' || check == '[' && c == ']') ){
+					return false;
+				}
+			}
+		}
+		if( !myStack.isEmpty() ){
```

Comment:
instead of lines 38-42, can we write 1 line return myStack.isEmpty()?
They use this term, "reduce boilerplate code". this is one such example, where we can write less lines of code.
But overall impressed with your approach and logic, & also handling all corner cases! keep up the good job!